### DISCUSSION TITLE
Unable to import CSV, even though the file is correct

### DISCUSSION THREAD

**Discussion by ilhamgum** [supabase/supabase#28501] [Questions] [answered]
So, I have this CSV file that I want to import, done checked the header and even the data inside of it, all of them.

![Screenshot 2024-08-09 141229](https://github.com/user-attachments/assets/3457dfe6-110e-4aa7-822d-007f164e79db)

Also, no error given on supabase, but after click "Import data", it gives me this error

`The data that you are trying to import is incompatible with your table structure`
![Screenshot 2024-08-09 141334](https://github.com/user-attachments/assets/5923d20c-1a44-41b0-8dee-921450bad501)

What should I do next? I have re-checked it several times, and ensure it does not contain any error on the data

**Comment by yeonjulee1005** [comment]
I want answer this discussion too.. 
also table column checked, and in my case like this..
<img width="657" alt="image" src="https://github.com/user-attachments/assets/f0689353-91a4-40d4-b695-0bb75ee19bd7">

**Comment by ghost** [comment]
same here...
<img width="734" alt="ss2024-08-09" src="https://github.com/user-attachments/assets/add30c0b-fb04-4ecf-8e17-c874085508cf">

**Comment by omaryasirm** [comment]
Facing same issue

**Comment by igorator** [comment]
I've got same issue too

**Comment by samyakjain0606** [comment]
I am also having same issue.

**Comment by taeyoon0137** [comment]
same issue +1

**Comment by GaryAustin1** [comment, marked as answer]
Looks like a fix is coming:
https://github.com/supabase/supabase/pull/28509

Hard refresh your browser to get the fix.

**Comment by chez23** [comment]
I also face the same problem
